apps/explorer/src/pages/transaction-result/TransactionView.tsx

```diff
@@ -310,6 +310,21 @@ export function TransactionView({
                             />
                         )}
 
+                        {transaction.checkpoint && (
+                            <section className="py-12">
+                                <TableHeader>Checkpoint Detail</TableHeader>
+                                <div className="pt-4">
+                                    <DescriptionItem title="Checkpoint Seq. Number">
+                                        <CheckpointLink
```

Comment:
nit: I guess this is more of a semantic thing, but technically `transaction.checkpoint` is the sequence number and not a `digest`. Maybe we should rename the `digest` prop on `CheckpointLink` to something like `digestOrSequenceNumber`

Reply:
@williamrobertson13 Should we change how its displaying also? Coz this is what it looks like when passing into that `CheckpointLink` component

<img width="463" alt="Screenshot 2023-03-23 at 1 30 38 PM" src="https://user-images.githubusercontent.com/127577476/227347256-612023c8-86ab-4b0c-a042-052f3cc663be.png">

Reply:
Actually I need to not truncate the value. Will make the change for the prop name

Reply:
oh in that case, I'd create a separate `CheckpointSequenceLink` 

```
// The third argument is the "formatter" which defaults to the address formatter
export const CheckpointSequenceLink = createInternalLink('checkpoint', 'sequenceNum', null);
```

Reply:
Probably need to fix the `truncate` logic, it seems to just add `...` even for strings that are way too short. But I will include that on later PR if that is a problem.

<img width="422" alt="Screenshot 2023-03-23 at 1 33 43 PM" src="https://user-images.githubusercontent.com/127577476/227348450-af50bd55-a70e-442a-9ef2-8e041cbd3700.png">